README.md

```diff
@@ -33,9 +33,10 @@
 | [John Hunter Matplotlib Summer Fellowship](https://www.numfocus.org/programs/john-hunter-technology-fellowship)| Yes |
 | [GNOME Internship Program](https://wiki.gnome.org/Internships)| Yes |
 | [Igalia Coding Experience Program](https://www.igalia.com/about-us/coding-experience)| Yes |
-
+| [NJACK Winter of Code ](https://njackwinterofcode.github.io/)| No***|
 *Interns must be currently enrolled or employed at a U.S. university or other research institution and must currently reside in, and be eligible to work in, the United States.
 ** OSoC is only open to Belgian students.
+*** NJACK is only open to NJACK students
```

Comment:
Hi, thanks for the PR. What do you mean by NJACK students? NJACK is the computer science club of IIT Patna and NWoC is open to everyone. Also there are swag rewards for participants. Also I think this should go in the `Open Source Competitions` section.

cc: @Akshat-Jain